src/main/java/org/minions/devfund/angela/battleship/Ocean.java

```diff
@@ -0,0 +1,177 @@
+package org.minions.devfund.angela.battleship;
+
+import java.util.Random;
+
+/**
+ * Manages ship's ocean.
+ */
+public class Ocean {
+    private static final int OCEAN_SIZE = 20;
+    private static final int SHIPS_QUANTITY = 13;
+    private Ship[][] ships;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+
+    /**
+     * Initializes an instance of {@link Ocean}.
+     */
+    public Ocean() {
+
+        ships = new Ship[OCEAN_SIZE][OCEAN_SIZE];
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        for (int i = 0; i < OCEAN_SIZE; i++) {
+            for (int j = 0; j < OCEAN_SIZE; j++) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+    }
+
+    /**
+     * Paces 13 ships randomly.
+     */
+    void placeAllShipsRandomly() {
+
+        final int fourLengthShip = 4;
+        final int sixLengthShip = 6;
+        final int nineLengthShip = 9;
+        Random random = new Random();
+        Ship[] shipsToPlace = new Ship[SHIPS_QUANTITY];
+        for (int i = 0; i < SHIPS_QUANTITY; i++) {
+            if (i == 0) {
+                shipsToPlace[i] = new BattleShip();
+            } else if (i == 1) {
+                shipsToPlace[i] = new BattleCruiser();
+            } else if (i < fourLengthShip) {
+                shipsToPlace[i] = new Cruiser();
+            } else if (i < sixLengthShip) {
+                shipsToPlace[i] = new LightCruiser();
+            } else if (i < nineLengthShip) {
+                shipsToPlace[i] = new Destroyer();
+            } else {
+                shipsToPlace[i] = new Submarine();
+            }
+        }
```

Comment:
Could you refactor this?
It is not a good practice to have a lot of if conditionals
Don't change using a switch
You should try another thing.